### PR TITLE
Pp1 5746 : replaced usage of valueOrError with errable

### PR DESCRIPTION
**Pull request by dibyanideb**

### CODE REVIEW

File: vajram/vajram-codegen/src/main/java/com/flipkart/krystal/vajram/codegen/VajramCodeGenerator.java [hunk 1]
```diff
@@ -871,38 +871,42 @@ private void generateDependencyResolutions(
           } else {
             String message =
                 ("A resolver ('%s') must not access an optional dependencyDef ('%s') directly."
-                        + "Use Optional<>, Errable<>, or DependencyResponse<> instead")
+                        + "Use Optional<>, ValueOrError<>, or DependencyResponse<> instead")
```
Comment (dibyanideb) [inline vajram/vajram-codegen/src/main/java/com/flipkart/krystal/vajram/codegen/VajramCodeGenerator.java:874]: this is not my changes but upon merging with main and resolving conflicts its showing as my diff

Comment (RamAnvesh) [reply vajram/vajram-codegen/src/main/java/com/flipkart/krystal/vajram/codegen/VajramCodeGenerator.java:874]: In main branch all usages of ValueOrError are replaced with Errable. Please do the same

Comment (dibyanideb) [reply vajram/vajram-codegen/src/main/java/com/flipkart/krystal/vajram/codegen/VajramCodeGenerator.java:874]: Done

**Comment by RamAnvesh** [comment]
@dibyanideb Please update PR title with description of the change

**Comment by dibyanideb** [comment]
merged same changes in PR-222 https://github.com/flipkart-incubator/Krystal/pull/222/files

**Pull request closed without merging**